app/controllers/projects_controller.rb

```diff
@@ -0,0 +1,14 @@
+class ProjectsController < ApplicationController
+def show 
+@projects = Project.find(params[:id])
+@contestants = @projects.contestants
```

Comment:
This works; a few notes: 
* If you're only finding one project instance, I would call it `@project` singular, to differentiate between a collection and a single instance. Since it's just a variable it doesn't matter really, but it will matter when you're juggling lots of variables! 
* You can also call `@project.contestants` in the view to access the association there instead of here, but either way is fine.